docs/src/data/component-status.yaml

```diff
@@ -601,3 +601,6 @@
   react: Released
   docs: Released
   group: Interaction
+- component: KeyValue
```

Comment:
Didn't we delete this?

Reply:
Yes. I deleted this yesterday. But apparently this job picks what's done for iOS and Android and because it detected this component in their libraries, it added it back here. I think it's ok to approve 🤔

Reply:
But anyway, it's not that this list is super up to date / reliable anyway 😬